src/views/v2/components/events-bar/views.php

```diff
@@ -33,7 +33,9 @@
 			class="tribe-events-c-view-selector__button"
 			data-js="tribe-events-view-selector-button"
 		>
-			<span class="tribe-events-c-view-selector__button-icon tribe-common-svgicon <?php echo sanitize_html_class( "tribe-common-svgicon--{$view_slug}" ); ?>"></span>
+			<span class="tribe-events-c-view-selector__button-icon">
+				<?php $this->template('components/icons/' . esc_attr( $view_slug ), [ 'classes' => ['tribe-events-c-view-selector__button-icon-svg'] ] ); ?>
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: 007b44287207e03f6ad5874c77e8f7a7</sub>